lnrpc/rpc.proto

```diff
@@ -1428,6 +1428,15 @@ message Channel {
     cooperatively closing with the delivery_address field set.
     */
     string close_address = 25;
+
+    /*
+    The amount that the initiator of the channel optionally pushed to the remote
+    party on channel open. This amount will be zero if the channel initiator did
+    not push any funds to the remote peer. If the initiator field is true, we
+    pushed this amount to our peer, if it is false, the remote peer pushed this
+    amount to us.
+    */
+    uint64 push_amount = 27;
```

Comment:
Should we call this `push_amount_sat` or at least mention in the comment that it's in satoshis?
It's an amount that's in the commitment transaction so it can never be mSat but maybe that's not obvious to every user.

Reply:
Definitely better to make it clear!